src/createAPI.js

```diff
@@ -5,8 +5,8 @@ import applyMiddleware from './applyMiddleware';
 const createAPI = (
     resources = {},
     middleware = [],
-    APINamespace = '',
-    fetchOptions = {}
+    APINamespace,
+    fetchOptions
```

Comment:
@TimeRaider generally it's a bug, because the documentation says completely different default values :)